pkg/controller/instance/instance_controller.go

```diff
@@ -113,18 +113,21 @@ func (r *Reconciler) SetupWithManager(
 		Complete(r)
 }
 
-// eventFilter ignores DeleteEvents for pipe-pods only (marked with task.PipePodAnnotation). This is due to an
-// inherent race that was described in detail in #1116 (https://github.com/kudobuilder/kudo/issues/1116)
-// tl;dr: pipe-task will delete the pipe pod at the end of the execution. this would normally trigger another
-// Instance reconciliation which might end up copying pipe files twice. we avoid this by explicitly ignoring
-// DeleteEvents for pipe-pods.
 func eventFilter() predicate.Funcs {
 	return predicate.Funcs{
 		CreateFunc: func(event.CreateEvent) bool { return true },
 		DeleteFunc: func(e event.DeleteEvent) bool {
+			// eventFilter ignores DeleteEvents for pipe-pods only (marked with task.PipePodAnnotation). This is due to an
+			// inherent race that was described in detail in #1116 (https://github.com/kudobuilder/kudo/issues/1116)
+			// tl;dr: pipe-task will delete the pipe pod at the end of the execution. this would normally trigger another
+			// Instance reconciliation which might end up copying pipe files twice. we avoid this by explicitly ignoring
+			// DeleteEvents for pipe-pods.
 			return !isForPipePod(e)
 		},
-		UpdateFunc:  func(event.UpdateEvent) bool { return true },
+		UpdateFunc: func(e event.UpdateEvent) bool {
+			// by comparing the Meta.Generation counter we effectively ignore Status sub-resource changes
+			return e.MetaNew.GetGeneration() != e.MetaOld.GetGeneration()
```

Comment:
This is the change ^^

Reply:
Do we actually want that? This would ignore status updates of Pods, Jobs, StatefulSets, etc. as well, which would lead to missing updates of the health of a resource, which means a plan may stay IN_PROGRESS for much longer, right?

Reply:
Might be ok to ignore status updates of `v1beta1.Instance` resources though...

Reply:
Yes, you're absolutely right! My intention was to **only** ignore `Instance.Status` changes.

Reply:
also nice catch @ANeumann82 ... the end result looks good.